pkg/controller/persistentvolume/index_test.go

```diff
@@ -203,6 +204,7 @@ func TestSort(t *testing.T) {
 	if err != nil {
 		t.Error("Unexpected error retrieving volumes by access modes:", err)
 	}
+	sort.Sort(byCapacity{volumes})
```

Comment:
does "byCapacity" definition belong in the unit test, if the unit test is the only place it is used?